config/prow/cluster/kubernetes-external-secrets_sa.yaml

```diff
@@ -3,8 +3,7 @@ kind: ServiceAccount
 metadata:
   annotations:
     eks.amazonaws.com/audience: sts.amazonaws.com
-    eks.amazonaws.com/role-arn: arn:aws:iam::609817409085:role/prow-ecr
+    eks.amazonaws.com/role-arn: xyz
```

Comment:
shouldn't we give this the correct role name?

Reply:
I was thinking if we could update this one with the external secrets PR  #62 getting merged?

Reply:
that works as well. Is #62 updated? Is it in a state to be reviewed?